hedera-node/hedera-app/src/test/java/com/hedera/node/app/fees/MonoGetTopicInfoUsageTest.java

```diff
@@ -61,25 +61,27 @@ class MonoGetTopicInfoUsageTest {
 
     private final Key key = A_COMPLEX_KEY;
     private final EntityNum topicEntityNum = EntityNum.fromLong(1L);
-    private final TopicID topicId =
-            TopicID.newBuilder().setTopicNum(topicEntityNum.longValue()).build();
+    private final com.hedera.hapi.node.base.TopicID topicId =
```

Comment:
Looks like `com.hederahashgraph.api.proto.java.TopicID` is removed from the imports. I assume you can simply import the new package.